docs/steps-elements.md

```diff
@@ -201,12 +201,55 @@ Allows to check if `:elementName` has a value that matches the `:matcher`.
 
 ---
 
+## `there is element ":elementName" containing ":matcher" text`
+
+Allows to check if `:elementName` contains a text that matches the `:matcher`.
```

Comment:
This is text matcher, you cannot pass `r:` in this step.

Add also some examples because after reading that, I still don't know how to use it